pam_authz/docker/run.dockerfile

```diff
@@ -12,9 +12,21 @@ RUN rm -f /etc/service/sshd/down
 RUN /etc/my_init.d/00_regen_ssh_host_keys.sh
 
 RUN apt-get update && apt-get install -y \
-    vim \
-    sudo && \
-    rm -rf /var/lib/apt/lists/*
+	vim \
+	wget \
+	gcc \
+	make \
+	sudo && \
+	rm -rf /var/lib/apt/lists/*
+
+RUN wget http://www.digip.org/jansson/releases/jansson-2.11.tar.gz && \
```

Comment:
Can you pluck the required files out of the build container instead of building the jansson lib here too?

Reply:
Done.